README.md

```diff
@@ -62,4 +62,35 @@ Validator validator = new FastaValidator(isr, SequenceType.DNA);
 ValidationResult result = validator.validate();
 
 assertTrue(result.isValid());
-```
\ No newline at end of file
+```
+
+### Using biovalidator in your application
+
+Maven dependency
+```xml
+<dependency>
+  <groupId>org.intermine</groupId>
+  <artifactId>biovalidator</artifactId>
+  <version>0.1.0</version>
+  <type>pom</type>
+</dependency>
+```
+
+Gradle dependency
+```groovy
+compile 'org.intermine:biovalidator:0.1.0'
+```
+
+### Using biovalidator as a command line utility
+Step 1: Get biovalidator Fat-Jar(either build from source or dirctly download from [bintray](https://bintray.com/intermineorg/biovalidator/biovalidator#files/org%2Fintermine%2Fbiovalidator%2F0.1.0))
+* Build from source : $ gradle createFatJar
+
+* Or Download fat-jar version of biovalidator from [bintray](https://bintray.com/intermineorg/biovalidator/biovalidator#files/org%2Fintermine%2Fbiovalidator%2F0.1.0)
+
+Setp 2: run as an executable jar file
+```shell
+java -jar biovalidator-0.1.0.jar --help
```

Comment:
please update this!

Change to be one of these:

# run this in your build/libs directory
> java -jar biovalidator-0.1.0.jar --help

OR

> java -jar build/libs/biovalidator-0.1.3.jar --help